integration/admin-client-jee/src/main/java/org/keycloak/admin/client/resource/RoleResource.java

```diff
@@ -133,6 +133,24 @@ public interface RoleResource {
     @Produces(MediaType.APPLICATION_JSON)
     List<UserRepresentation> getUserMembers(@QueryParam("first") Integer firstResult,
             @QueryParam("max") Integer maxResults);
+
+    /**
+     * Get role members.
+     * <p/>
```

Comment:
This is closing an HTML paragraph that doesn't exist.
```suggestion
```

Reply:
:+1: I have also changed the other methods with the same tag. I just copy-pasted it. :smile: